cli/packages/graphcool-cli-core/src/commands/import/Validator.ts

```diff
@@ -59,6 +59,14 @@ export class Validator {
         )
       )
     },
+    Json: (str) => {
```

Comment:
Hi @adrienbarreau 

I don't understand why you get an object in str, you should get a string on Json format.

If you have an object, the test should fail...
Did I miss something?

Reply:
@GaelGRIFFON Your code is assuming that the input to the `Json` function will be a string, but actually, it's an object. That's because the exporter already exports it as a javascript object.
I think you can just do this:
```
Json: obj => typeof obj === 'object' && obj !== null,
```

Reply:
OK got it!
Thx you